### PR TITLE
bump timeouts to 50 to avoid flaps

### PR DESCRIPTION
**Pull request by Raynos**
r: @kriskowal @rf

### CODE REVIEW

File: hyperbahn/cut.sh [hunk 1]
```diff
@@ -67,5 +67,5 @@ git push origin master --tags
 git push -f origin dev_hyperbahn
 
 git archive --prefix=package/ --format tgz dev_hyperbahn >package.tgz
-${NPM:-npm} publish package.tgz --tag "${NPM_TAG:-alpha}"
+${NPM:-npm} publish package.tgz --tag "${NPM_TAG:-latest}"
```
Comment (kriskowal) [inline hyperbahn/cut.sh]: this will make a sad @jcorbin.

Comment (kriskowal) [inline hyperbahn/cut.sh]: please remove so we can green stamp the rest

Comment (Raynos) [inline hyperbahn/cut.sh]: No. Hyperbahn should always be published as latest.

Comment (kriskowal) [inline hyperbahn/cut.sh]: Separable PR.

**Pull request merged**